_includes/head.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  {% assign title = page.title | default: site.title | escape %} {% assign description = page.description | default:
+  site.description | strip_html | strip_newlines | truncate: 160 %} {% assign image = page.image | default:
+  site.author.image %} {% if page.name and page.collection == 'projects' %} {% assign title = page.name | escape %} {%
+  endif %}
+
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="{{ title }}" />
+  <meta property="og:description" content="{{ description }}" />
+  <meta property="og:image" content="{{ image }}" />
+
+  <title>{{ title }}</title>
+  <meta name="description" content="{{ description }}" />
+
+  <link rel="shortcut icon" type="image/x-icon" href="{{ '/assets/favicon.ico' | relative_url }}" />
+
+  <!-- Theme style -->
+  <script src="{{ '/assets/js/theme.js' | relative_url }}"></script>
+
+  <!-- Fork Awesome CDN -->
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+    integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+    crossorigin="anonymous"
+  />
+
+  <!-- Bootstrap CSS CDN -->
+  <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+    integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+    crossorigin="anonymous"
+  />
```

Comment:
from where is the integrity from? do we need to update them when we bump up the versions?

Reply:
![image](https://github.com/randovania/randovania.github.io/assets/2979303/d04a3ef0-c4cb-4a4a-ae98-ed5143c1df54)
i literally just copied the cdn urls and they came with the field

Reply:
sooo... what do we need to do then if we bump up the versions?

Reply:
copy the cdn urls from their sites